pennylane/qchem/basis_set.py

```diff
@@ -95,7 +97,11 @@ def atom_basis_data(name, atom):
     # for dxy, dxz, dyz, dxx, dyy, dzz, respectively:
     d = [(1, 1, 0), (1, 0, 1), (0, 1, 1), (2, 0, 0), (0, 2, 0), (0, 0, 2)]
 
-    basis = basis_sets[name][atom]
+    if load_data:
+        basis = load_basisset(name, atom)
+    else:
```

Comment:
Do we raise a warning to suggest users use the `flag` in case the basis set they requested doesn't exist but exists in `bse`?

Reply:
Added it to the molecule class which is the user-facing.